merge.go

```diff
@@ -111,7 +111,7 @@ func mergeSegmentBases(segmentBases []*SegmentBase, drops []*roaring.Bitmap, pat
 
 func MergeToWriter(segments []*SegmentBase, drops []*roaring.Bitmap,
```

Comment:
Does this API need to be exported?

Reply:
no, it does not find its usage in bleve
i will unexport it